components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.html

```diff
@@ -8,7 +8,7 @@
             type="error"
             timeout="5"
             (dismissed)="hideDownloadError()">
-            Failed to install profile. Please try again later.
+            You are not authorized to create compliance profiles.
```

Comment:
❓ Is that the only way that this could go wrong? I'd think that we might inspect the actual error response and perhaps look at its code? Or use its message.

Reply:
@susanev Can you give me your thoughts on that?

Reply:
oh sorry, i dont know that. well need to ask in superteam, or work with @vjeffrey

Reply:
yes, other things can go wrong -- ill carve out some time to look at the err msgs returned today

Reply:
so, other things can go wrong, like getting an invalid request or a 404, but those things will likely not happen via the ui call..

Reply:
Not even if another user has removed the profile using the UI in the meantime? I think we should bubble up the actual error message like we do in other places... 🤔

Reply:
you can delete an available profile